token/cli/src/main.rs

```diff
@@ -7425,4 +7453,43 @@ mod tests {
                 .unwrap()
         );
     }
+
+    #[tokio::test]
+    #[serial]
+    async fn metadata_pointer() {
+        let (test_validator, payer) = new_validator_for_test().await;
+        let config =
+            test_config_with_default_signer(&test_validator, &payer, &spl_token_2022::id());
+        let program_id = &spl_token_2022::id();
+        let token_path = NamedTempFile::new().unwrap();
+        let metadata_address = Pubkey::new_unique();
+
+        let result = process_test_command(
+            &config,
+            &payer,
+            &[
+                "spl-token",
+                CommandName::CreateToken.into(),
+                token_path.path().to_str().unwrap(),
+                "--program-id",
+                &program_id.to_string(),
+                "--metadata-address",
+                metadata_address.to_string().as_str(),
```

Comment:
nit for consistency with `program_id`
```suggestion
                &metadata_address.to_string(),
```

Reply:
Here we expect a string slice, that's why the `metadata_address.to_string().as_str()`